package.json

```diff
@@ -31,6 +32,9 @@
     "@types/jasmine": "~3.6.0",
     "@types/node": "^12.11.1",
     "codelyzer": "^6.0.0",
+    "commitizen": "^4.2.3",
+    "git-cz": "^4.7.6",
+    "husky": "^5.1.3",
```

Comment:
I think you've added husky to enforce cz on each commit but the hook is not configured.